projects/ng-core-tester/src/app/app.component.ts

```diff
@@ -16,7 +16,8 @@
 */
 import { Component, OnInit } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
-import { CoreConfigService } from '@rero/ng-core';
+import { CoreConfigService, RecordService } from '@rero/ng-core';
```

Comment:
RecordService was imported from /public-apis/record-service and it caused the misfunction.